app.js

```diff
@@ -0,0 +1,72 @@
+const tipAmount = document.querySelector(".tip-amount");
+const totalAmount = document.querySelector(".total-amount");
+const inputBill = document.querySelector(".bill");
+const selectBtn = document.querySelectorAll("button");
```

Comment:
배열이거나 컬렉션인 경우 변수명에 복수형을 의미하는 s를 붙여주면 좋을것 같습니다.
```suggestion
const selectBtns = document.querySelectorAll("button");
```